avro-serializer/src/main/java/io/confluent/kafka/formatter/AvroMessageFormatter.java

```diff
@@ -143,6 +155,20 @@ public void init(Properties props) {
 
   @Override
   public void writeTo(ConsumerRecord<byte[], byte[]> consumerRecord, PrintStream output) {
+    TimestampType timestampType = consumerRecord.timestampType();
+    if (printTimestamp) {
+      try {
+        if (timestampType != TimestampType.NO_TIMESTAMP_TYPE) {
+          output.write(String.format("%s:%d",
+                  timestampType, consumerRecord.timestamp()).getBytes(StandardCharsets.UTF_8));
+        } else {
+          output.write("NO_TIMESTAMP".getBytes(StandardCharsets.UTF_8));
+        }
+        output.write(keySeparator);
```

Comment:
BTW, `ConsoleConsumer` allows for `--print.value=false` and checks if `printKey || printValue` before printing a separator